docker-hbase/Dockerfile

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Intel Corporation
```

Comment:
@jelyoussefi Nice work. Only thing is can you update the file headers with up-to-date header and use the same for the other files, docker-hbase/scripts/entrypoint.sh, etc.